org-brain.el

```diff
@@ -62,24 +64,94 @@ This will be used by `org-brain-new-child'."
   :group 'org-brain
   :type '(string))
 
+;;; Utils
+(defun flatten (obj)
```

Comment:
This is just Common Lisp code that I had from a CL project. It probably would be safer to not muddy the namespace. I'll prepend it with org-brain. Thanks.